### PR TITLE
Return Of The Pibble

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

This PR Rebases one of Nyanotrasen's funniest features: The Pibble.
I mean "Lab Mix", according to the description. Lab Mixes are comically muscular dogs famous for their ability to attack and destroy Felinids, Mailmen, and more recently Harpies. Watch out, there is also a rare variant of Lab-Dachshund Mix, colloquially known as the "Ventbull", which can occasionally be found roaming the station's vents. Please keep your children, Felinids, and Birbs at a safe distance.  

# Changelog

:cl:
- add: Lab Mixes have been added to the game as a new random animal. Be sure to keep Felinids and Harpies away from them.

### CODE REVIEW

**Comment by Elijahrane** [comment]
I had new AI, utility, content etc planned for pibbles in my big ghostie / NPC feature arc if you don't mind holding off a bit

File: Resources/Prototypes/Datasets/Names/pitbull.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - Cuddles
+  - Killer
+  - Paisley
+  - Echo
+  - Sweetie
+  - Tutu
+  - Reaper
+  - Peaches
+  - Cujo
+  - Cuddles
```
Comment (Elijahrane) [inline Resources/Prototypes/Datasets/Names/pitbull.yml:25]: Cuddles is in here twice

Comment (OldDanceJacket) [reply Resources/Prototypes/Datasets/Names/pitbull.yml:25]: Cuddles is a good enough name to be seeded to appear more

**Comment by OldDanceJacket** [comment]
> I had new AI, utility, content etc planned for pibbles in my big ghostie / NPC feature arc if you don't mind holding off a bit
small updates like this can go through. 

Your planned feature isn't dependent on pibbles, and pibbles aren't dependent on yours, it should be fine to introduce pibbles back

**Review by OldDanceJacket — APPROVED** [review]

File: Resources/Textures/Mobs/Pets/pitbull.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "version": 1,
+  "size":
+  {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Hyenh",
```
Comment (DangerRevolution) [inline Resources/Textures/Mobs/Pets/pitbull.rsi/meta.json:9]: Who?

File: Resources/Textures/Mobs/Pets/ventbull.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "version": 1,
+  "size":
+  {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Hyenh",
```
Comment (DangerRevolution) [inline Resources/Textures/Mobs/Pets/ventbull.rsi/meta.json:9]: Who?

File: Resources/Prototypes/Entities/Mobs/NPCs/dogs.yml [hunk 4]
```diff
@@ -0,0 +1,120 @@
+- type: entity
```
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Mobs/NPCs/dogs.yml:1]: Needs a language, especially as a funny pet is a likely target for Cognizine

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

File: Resources/Textures/Mobs/Pets/pitbull.rsi/meta.json (hunk 2 above)
Comment (DEATHB4DEFEAT) [reply Resources/Textures/Mobs/Pets/pitbull.rsi/meta.json:9]:
```suggestion
  "copyright": "Hyenh#6078 (313846233099927552)",
```

File: Resources/Textures/Mobs/Pets/ventbull.rsi/meta.json (hunk 3 above)
Comment (DEATHB4DEFEAT) [reply Resources/Textures/Mobs/Pets/ventbull.rsi/meta.json:9]:
```suggestion
  "copyright": "Hyenh#6078 (313846233099927552)",
```

File: Resources/Prototypes/Datasets/Names/pitbull.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Datasets/Names/pitbull.yml:1]: Alphabetically sort this :)

**Review by DEATHB4DEFEAT — APPROVED** [review]
As long as Walter's not a pitbull

**Pull request merged**